fields/class-swp-acf-color-palette-v5.php

```diff
@@ -195,33 +195,17 @@ function render_field( $field ) {
 		 * @return n/a
 		 */
 		function input_admin_enqueue_scripts() {
-			global $wp_scripts;
-
 			$url = $this->settings['url'];
 			$version = $this->settings['version'];
 			$min = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
 
-			// register if not already (on front end)
```

Comment:
Can you restore this part please? That is no relation with the way to load `wp-color-picker-alpha` ;)